tests/common/project_testing.py

```diff
@@ -66,7 +70,7 @@ def __init__(self, *args, **kwargs):
         self.maxDiff = None
     
     def tearDown(self):
-        self.simlab.destroySims()
+        pass#self.simlab.destroySims()
 
```

Comment:
Was this intentional (commented out part)?

Reply:
Nope, will fix.